app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -172,9 +172,9 @@ fun SettingGroupLabel(text: String, modifier: Modifier = Modifier) {
 @Composable
 fun Setting(
     label: String,
-    description: String,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
+    description: String = "",
```

Comment:
Oh ok
In some projects I know that this isn't always something that's available so I did it on autopilot honestly. But since you want to have the consistency, I reverted this change.